docs/source/user_guide/runtime-conf.md

```diff
@@ -41,7 +41,8 @@ elyra-metadata install runtimes --schema_name=kfp \
        --cos_endpoint=http://minio-service.kubeflow:9000 \
        --cos_username=minio \
        --cos_password=minio123 \
-       --cos_bucket=test-bucket
+       --cos_bucket=test-bucket \
+       --tags=("my-kfp")
```

Comment:
btw this command doesn't work 🙄
@kevin-bates is it possible to create a runtime with tags field from CLI?